protostar/commands/test/test_command.py

```diff
@@ -26,6 +33,32 @@
 from protostar.utils.protostar_directory import ProtostarDirectory
 from protostar.utils.starknet_compilation import CompilerConfig, StarknetCompiler
 
+from protostar.self.cache import CacheUtil
+
+
+def read_targets_from_cache(cache_util, targets) -> Optional[list]:
```

Comment:
This function should be unit tested.

Reply:
ok

Reply:
> This function should be unit tested.

I think your comments deny each other. If we resign from considering targets when detecting the test cases, we'll rather remove this function completely. In such a case, we won't need unit tests.

Reply:
> Optionally, we could evaluate possible globs before calling this, but I do not mind abandoning this.

It could be tricky to do. Targets consist of two globs, `file_glob::test_case_glob`.

> I think your comments deny each other. If we resign from considering targets when detecting the test cases, we'll rather remove this function completely. In such a case, we won't need unit tests.

Yep, if this function is removed, there's no point creating a unit test.